### DISCUSSION TITLE
How can I ignore certain error codes?

### DISCUSSION THREAD

**Discussion by tibbe** [Tufin/oasdiff#626] [Q&A]
For example, our API generator makes it safe to add new members to a `oneOf` so I'd like to disable (or make a warning out of) `request-body-one-of-added`. Is this currently possible?

This is a common feature of linters in general.

(I don't want to list specific routes where this error is OK. I want to suppress it everywhere.)

**Comment by reuvenharrison** [comment]
Perhaps you can customize the error levels of the specific check/s to INFO?
See: https://github.com/Tufin/oasdiff/blob/main/docs/BREAKING-CHANGES.md#customizing-severity-levels

**Reply by reuvenharrison** [reply]
Hi @tibbe 
I decided to add a new custom check level `none` which can be used to disable a check completely.
Please see the documentation here: https://github.com/Tufin/oasdiff/blob/main/docs/BREAKING-CHANGES.md#customizing-severity-levels
I hope that answers your requirement.
Reuven
